hack/fetch_ext_bins.sh

```diff
@@ -26,17 +26,17 @@ if [[ -n "${TRACE}" ]]; then
   set -x
 fi
 
-k8s_version=1.19.2
-goarch=amd64
-goos="unknown"
+k8s_version=1.22.0
```

Comment:
we are hardcoding this here, and is it based on the value we use somewhere or?

Reply:
okay, we pinned k8s version to this everywhere (tiltfile, e2e), I am fine with it.